its/src/test/java/com/sonar/it/scanner/msbuild/sonarcloud/IncrementalPRAnalysisTest.java

```diff
@@ -62,7 +63,7 @@ void master_emptyCache() throws IOException {
 
   @Test
   void prWithoutChanges_producesUnchangedFilesWithAllFiles() throws IOException {
-    var context = AnalysisContext.forCloud(SONARCLOUD_PROJECT_KEY, basePath, PROJECT_NAME);
```

Comment:
There are still usages of `@TempDir basePath` in this file. Do you plan to remove all of these? It seems a bit unfortunate to have two different ways of dealing with the temp directory.

Reply:
SCAN4NET-373 will do the actual migration to AnalysisContext for this class, and then the `@TempDir` will be unused and can be removed.